README.md

```diff
@@ -2,10 +2,62 @@
+
+1. Go to the [GitHub repo](https://github.com/NYCPlanning/ae-zoning-api) for Zoning API
+2. Follow the steps there to clone the project, run the API, and run its database using docker compose
+3. Leave that Zoning API and its database running in a separate terminal, and navigate back to where you cloned this repo.
+4. Back in your cloned copy of this repo, create a file called `.env` in the root folder of the project and copy the contents of `sample.env` into that new file. The default value in `sample.env` should set the Zoning API URL variable to the correct value for your locally running instance of Zoning API
+
+### Run the project
+Finally, to run this project locally:
+```
+npm run dev
+```
+
+## Pulling data from [Zoning API](https://github.com/NYCPlanning/ae-zoning-api)
+This app relies heavily on data from the Zoning API backend application. It makes use of some front end libraries to make pulling data and managing the state of that data easier:
+
+* [Axios](https://axios-http.com/docs/intro) - A framework-agnostic HTTP client for javascript and typescript applications
+* [TanStack Query](https://tanstack.com/query/v4) - This is a general purpose asynchronous state management library for React. You can read more about what problems TanStack Query strives to solve [here](https://tanstack.com/query/v4/docs/react/overview#motivation). It provides useful state management such as caching and managing loading states.
+* [Kubb](https://www.kubb.dev/introduction) - A code generation library that reads [OpenAPI specification](https://www.openapis.org/) documents and generates typescript "client" libraries (sometimes also referred to as "SDKs"). It features a set of plugins to generate code for various JS/TS libraries, including Axios and TanStack Query. By pointing Kubb at the [OpenAPI document for Zoning API](https://github.com/NYCPlanning/ae-zoning-api/blob/main/openapi/openapi.yaml), we can avoid writing a lot of boilerplate code.
+
+### Regenerating the Zoning API client code
+When the OpenAPI spec file for Zoning API changes, we will need to regenerate the client code in this repo. To do that, simply run `npm run generate`. Kubb is configured to put all generated code in `/src/gen`. Note that even though that code is generated, we do want to commit it and includes updates in PRs as necessary. A developer should **never** make manually changes or additions to code in the `/src/gen` folder. It should be purely what is outputted by Kubb.
+
+> You may see an error in your terminal after running `npm run generate` referring to the "path" argument being undefined. The root cause of this is still being looked into but it doesn't seem to affect the generated code so you can safely ignore it for now.
```

Comment:
Kubb has since had a few updates. We should no longer see this message.

```suggestion
```

Reply:
I didn't notice the change, good to know!